common-util/src/main/java/org/sunbird/common/models/util/JsonKey.java

```diff
@@ -697,6 +697,24 @@
   public static final String SUNBIRD_USER_PROFILE_FIELD_DEFAULT_VISIBILITY =
       "sunbird_user_profile_field_default_visibility";
   public static final String DEFAULT_PROFILE_FIELD_VISIBILITY = "defaultProfileFieldVisibility";
+  public static final String COURSE_BATCH_NOTIFICATIONS_ACTIVE = "course_batch_notification_active";
+  public static final String BATCH_END_DATE = "batchEndDate";
+  public static final String BATCH_START_DATE = "batchStartDate";
+  public static final String TEMPLATE_NAME = "templateName";
+  public static final String BATCH_MENTOR_ENROL = "batchMentorEnrol";
+  public static final String BATCH_LEARNER_ENROL = "batchLearnerEnrol";
+  public static final String COURSE_INVITATION = "Course Invitation";
+  public static final String BATCH_LEARNER_UNENROL = "batchLearnerUnenrol";
+  public static final String BATCH_MENTOR_UNENROL = "batchMentorUnenrol";
+  public static final String BATCH_NAME = "batchName";
+  public static final String UNENROLL_FROM_COURSE = "Unenrolled from Course";
+  public static final String OPEN_BATCH_LEARNER_UNENROL = "openBatchLearnerUnenrol";
```

Comment:
use same terms , for learner we are using term as Participant.

Reply:
okay